src/components/Gallery/Images.jsx

```diff
@@ -1,9 +1,9 @@
-export default function Images({ source, onClick, className }) {
+export default function Images({ source, onClick }) {
   const IMAGE_PATH = "/images/";
 
   return (
     <img
-      className={className}
+      className="__image"

```

Comment:
you need to define the entire scss compiled:

```suggestion
      className=".gallery-thumbnail__image"
```

Remeber scss will concatenate selectors if you do it:
```scss
.main {
  &__item { }
}
```

the result will be `main__item`